localization/xliff/vscode-mssql.xlf

```diff
@@ -1539,6 +1539,9 @@
     <trans-unit id="mssql.query.executionTimeout">
       <source xml:lang="en">An execution time-out of 0 indicates an unlimited wait (no time-out)</source>
     </trans-unit>
+    <trans-unit id="mssql.resultsGrid.autoSizeColumns">
+      <source xml:lang="en">Auto size the columns width on initial results. Could have performance problems with a large number of columns or large cells</source>

```

Comment:
In your testing, did you find thresholds where this starts seeing performance issues?  We should know more about when we start hitting issues with this enabled.